ts/DataGrid/Cell.ts

```diff
@@ -58,24 +55,18 @@ class DataGridCell {
     /**
      * The column of the cell.
      */
-    public column: DataGridColumn;
+    public column: Column;
 
     /**
      * The row of the cell.
      */
-    public row: DataGridRow;
+    public row: Row;
 
     /**
      * The raw value of the cell.
      */
     public value: DataTable.CellType;
 
```

Comment:
What is `value` for other types of `Cell`s than `TableCell`s? Shouldn't we move this to `TableCell`?